lib/northbound.c

```diff
@@ -1817,6 +1817,16 @@ int nb_oper_data_iterate(const char *xpath, struct yang_translator *translator,
 
 		/* Find the list entry pointer. */
 		nn = dn->schema->priv;
+		if (!nn->cbs.lookup_entry) {
```

Comment:
Question: Shouldn't we validate all operations? Like nb_node_validate_cbs().

Reply:
@ton31337 I'm not sure I understand the question. This is already validated in nb_node_validate_cbs. This callback is not required for config-only data so the validation is passed.

Reply:
Gotcha.